src/connection.c

```diff
@@ -87,6 +87,12 @@ static void ConnFree(Connection *conn)
 
     LIST_REMOVE(conn, entries);
 
+#ifdef SSL
```

Comment:
```suggestion
#ifdef HAVE_TLS
```

I don't know if `SSL` was defined somewhere. Otherwise, it'll be a leak probably. Just fyi, when you need, you can manually trigger daily CI for your branch here: https://github.com/RedisLabs/redisraft/actions/workflows/daily.yml